FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+Drafts.swift

```diff
@@ -8,53 +8,67 @@
 
 // MARK: - Drafts
 extension ComposeViewController {
-    @objc internal func startDraftTimer() {
-        saveDraftTimer = Timer.scheduledTimer(withTimeInterval: 10.0, repeats: true) { [weak self] _ in
+    @objc func startDraftTimer() {
+        saveDraftTimer = Timer.scheduledTimer(withTimeInterval: 5.0, repeats: true) { [weak self] _ in
             self?.saveDraftIfNeeded()
         }
         saveDraftTimer?.fire()
     }
 
-    @objc internal func stopDraftTimer() {
+    @objc func stopDraftTimer(withSave: Bool = true) {
+        guard saveDraftTimer != nil else { return }
+
         saveDraftTimer?.invalidate()
         saveDraftTimer = nil
-        saveDraftIfNeeded()
+
+        if withSave {
+            saveDraftIfNeeded()
+        }
     }
 
-    private func shouldSaveDraft() -> Bool {
-        // https://github.com/FlowCrypt/flowcrypt-ios/issues/975
-        return false
-//        let newDraft = ComposedDraft(email: email, input: input, contextToSend: contextToSend)
-//        guard let oldDraft = composedLatestDraft else {
-//            composedLatestDraft = newDraft
-//            return true
-//        }
-//        let result = newDraft != oldDraft
-//        composedLatestDraft = newDraft
-//        return result
+    private func createDraft() -> ComposedDraft? {
```

Comment:
Draft is auto-saved when user taps back button - https://github.com/FlowCrypt/flowcrypt-ios/blob/feature/issue-1813-drafts/FlowCrypt/Controllers/Compose/Extensions/ComposeViewController%2BSetup.swift#L151

And it's also automatically saved each 5 seconds

Reply:
Thanks. I've tested it. Looks a little strange this behavior `Draft is auto-saved when user taps back button`.

A user can't leave a screen while a draft is saving. Firstly, I thought I didn't tap on the back button. 
What about a case when the saving process will take around 20 seconds? All this time a user will try to tap on the back button and will not understand why he can't leave a screen. (As it was for me during 5-6 seconds. I thought the app is frozen). It would be nice to have some progress bar or notification about saving a draft.

Reply:
I agree with @DenBond7.
I also thought app was frozen when I tap back button. I think we should show progress bar while saving draft.

Reply:
Or just save it in the background, with automatic delay and repeat on network error or error 5xx. Until the app is killed, of course.

Reply:
On simulator draft saving usually takes less than 1 second, so I didn't add any progress bar, but on device it takes several seconds for me too.
I'll try to find why it happens to make saving on device faster and will also run saving in background, so it won't affect user experience.